components/filter/README.md

```diff
@@ -1,51 +1,194 @@
+  });
+</script>
+<!-- docs: end hidden content -->
 <d2l-filter>
-  // Dimensions...
+	<d2l-filter-dimension-set key="course" text="Course">
+		<d2l-filter-dimension-set-value key="art" text="Art"></d2l-filter-dimension-set-value>
+		<d2l-filter-dimension-set-value key="astronomy" text="Astronomy" selected></d2l-filter-dimension-set-value>
+		<d2l-filter-dimension-set-value key="biology" text="Biology"></d2l-filter-dimension-set-value>
+		<d2l-filter-dimension-set-value key="chemistry" text="Chemistry"></d2l-filter-dimension-set-value>
+		<d2l-filter-dimension-set-value key="drama" text="Drama"></d2l-filter-dimension-set-value>
+		<d2l-filter-dimension-set-value key="english" text="English"></d2l-filter-dimension-set-value>
+	</d2l-filter-dimension-set>
+	<d2l-filter-dimension-set key="role" text="Role">
+		<d2l-filter-dimension-set-value key="admin" text="Admin"></d2l-filter-dimension-set-value>
+		<d2l-filter-dimension-set-value key="instructor" text="Instructor"></d2l-filter-dimension-set-value>
+		<d2l-filter-dimension-set-value key="student" text="Student"></d2l-filter-dimension-set-value>
+	</d2l-filter-dimension-set>
+	<d2l-filter-dimension-set key="semester" text="Semester" selection-single>
+		<d2l-filter-dimension-set-value key="fall" text="Fall"></d2l-filter-dimension-set-value>
+		<d2l-filter-dimension-set-value key="winter" text="Winter" selected></d2l-filter-dimension-set-value>
+		<d2l-filter-dimension-set-value key="spring" text="Spring"></d2l-filter-dimension-set-value>
+		<d2l-filter-dimension-set-value key="summer" text="Summer"></d2l-filter-dimension-set-value>
+	</d2l-filter-dimension-set>
 </d2l-filter>
-```
+``` 
 
-**Properties**
+### Single Vs Multi Dimensional 
+A filter can be a single dimension (like picking from a list of departments) or offer multiple dimensions (filter by role, or department, or something else). Single-dimension filters can be used side-by-side to promite filters that are more commonly used. while tucking less-used filters into a multi-dimensional filter
+<!-- docs: demo code name:d2l-filter autoSize:false align:start size:large -->
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/filter/filter.js';
+  import '@brightspace-ui/core/components/filter/filter-dimension-set.js';
+  import '@brightspace-ui/core/components/filter/filter-dimension-set-value.js';
+</script>
+<!-- docs: start hidden content -->
+<script>
+  window.addEventListener('load', function () {
+    var demoElem = document.querySelector('#demo-element');
+    if (!demoElem.hasAttribute('data-first-load')) return;
+    setTimeout(() => {
+		var filter = document.querySelector('d2l-filter');
+		filter.opened = true;
+    }, 100);
+  });
+</script>
+<style>
+	.flex-div {
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+	}
+</style>
+<!-- docs: end hidden content -->
+<div class="flex-div">
+	Single Dimension
+	<d2l-filter>
+		<d2l-filter-dimension-set key="course" text="Course">
```

Comment:
Should this be something different like departments (to match the text about promoting some filters and hiding others away)?